config/routes.rb

```diff
@@ -11,12 +11,6 @@
   resources :named_contacts, only: :create
 
   namespace :anonymous_feedback do
-    get 'problem_reports', to: redirect {|p, req| req.params[:path] ? "/anonymous_feedback?path=" + req.params[:path] : "/anonymous_feedback"}
-
-    namespace :problem_reports do
-      get :explore, to: redirect("/anonymous_feedback/explore"), format: false
-    end
```

Comment:
Kibana log search for this endpoint:
<img width="674" alt="screen shot 2016-09-13 at 14 16 15" src="https://cloud.githubusercontent.com/assets/1370570/18717211/256148de-8017-11e6-894d-3121ec2ad37c.png">